test/end-to-end/core.sh

```diff
@@ -109,6 +109,21 @@ os::cmd::expect_success "curl -f http://${DOCKER_REGISTRY}/healthz"
 os::cmd::expect_success "dig @${API_HOST} docker-registry.default.local. A"
 registry_pod="$(oc get pod -n default -l deploymentconfig=docker-registry --template='{{(index .items 0).metadata.name}}')"
 
+os::log::info "Configure registry to disable mirroring"
+os::cmd::expect_success "oc project '${CLUSTER_ADMIN_CONTEXT}'"
+os::cmd::expect_success 'oc env -n default dc/docker-registry REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_MIRRORPULLTHROUGH=false'
+os::cmd::expect_success 'oc rollout status dc/docker-registry'
+os::log::info "Registry configured to disble mirroring"
+
+os::log::info "Check the image based on a non-local base image."
```

Comment:
Could this be more verbose? E.g.: *Verify that an image based on a remote image can be pushed to the same image stream while pull-through enabled*